app/forms/questionnaire_form.py

```diff
@@ -16,12 +16,14 @@
 
 
 class QuestionnaireForm(FlaskForm):
-    def __init__(self, schema, question_schema, answer_store, metadata, location, **kwargs):
+    def __init__(
+        self, schema, question_schema, answer_store, metadata, location, **kwargs
+    ):
         self.schema = schema
         self.question = question_schema
         self.answer_store = answer_store
         self.metadata = metadata
-        self.location = location,
+        self.location = (location,)
```

Comment:
Is there a reason why this is a tuple?

Reply:
Not sure how that got there, but have removed it now.